README.md

```diff
@@ -375,6 +375,19 @@ Note: `sudo` is needed above for the redirection server (to bind port 80).
 
 A valid email address is necessary for the creation of the certificate, and is important to get notifications from the Certificate Authority - in case the certificate is about to expire, etc.
 
+## Supporting CORS
+
+When accessing DefraDB through a frontend interface, you may be confronted with a CORS error. That is because, by default, DefraDB will not have any allowed origins set. To specify which origins should be allowed to access your DefraDB endpoint, you can specify them when starting the database:
+```shell
+defradb start --allowedorigins=https://yourdomain.com
```

Comment:
suggestion:

How about just `origins`. Mostly because we don't have a `blockedorigins` or any other option like that. Also `allowedorigins` is less readable, perhaps `allowedOrigins` if you want to stick with the "allowed" word.

```suggestion
defradb start --allowedorigins=https://yourdomain.com
```

Reply:
The thing with `--origins` is that it's a little vague and the http header is called `Allowed-Origins`. I feel like it makes more sense to have the full representation here as well. `allowedOrigins` could be nicer.

Reply:
nitpick: I have a very minor preference for `allowed-origins`, but only if it we dont already camel case any other existing multi-word params/flags

Reply:
I do like `allowed-origins`, would prefer that over `allowedOrigins`

Reply:
personally i'd like `--api.corsdomain` or `http.corsdomain` depending on which top-level key we want to represent the http system

Reply:
I'd rather we stay closer to what it represents. `allowed-origins` is quite good.

Reply:
so the fully qualified key from the config's perspective would be `api.allowed-origins`, and as an env. variable it would be `DEFRA_API_ALLOWED_ORIGINS` perhaps. i'm not sure yet about if it's necessary to quote such a key with a `-` in Yaml  - it might have to be

```yaml
[api]
"allowed-origins": ...
```

Reply:
I don't think it is according to what I see here https://www.cloudbees.com/blog/yaml-tutorial-everything-you-need-get-started